src/renderer/components/editor/EditorPreview.vue

```diff
@@ -138,6 +141,9 @@ onMounted(() => {
       display: flex;
       align-items: center;
       gap: var(--spacing-xs);
+      .button {
```

Comment:
Since the application imitates the native OS, the pointer cursor appears only on links. So you need to remove this styles.

Reply:
ok, but it's currently showing 'n-resize'. Is it better to change to default?

Reply:
It seems that if you go from the panel to that button, the mouse pointer becomes the wrong style.

Reply:
This seems to be a problem with `interactjs`. I'll take a look at it later.

Reply:
ok,add `cursor: default;` will fixed. Now like this. 
![GIF 2022-7-23 13-13-41](https://user-images.githubusercontent.com/17669599/180591597-4079c49c-193b-4fde-9420-6da4fba281dc.gif)

Reply:
> ok, but it's currently showing 'n-resize'. Is it better to change to default?
Yes, the `cursor: default` fix this bug.